src/test/java/com/mongodb/hibernate/internal/translate/mongoast/command/AstDeleteCommandTests.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2025-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.hibernate.internal.translate.mongoast.command;
+
+import static com.mongodb.hibernate.internal.translate.mongoast.filter.AstComparisonFilterOperator.EQ;
+
+import com.mongodb.hibernate.internal.translate.mongoast.AbstractAstNodeTests;
+import com.mongodb.hibernate.internal.translate.mongoast.AstLiteralValue;
+import com.mongodb.hibernate.internal.translate.mongoast.filter.AstComparisonFilterOperation;
+import com.mongodb.hibernate.internal.translate.mongoast.filter.AstFieldOperationFilter;
+import com.mongodb.hibernate.internal.translate.mongoast.filter.AstFilterFieldPath;
+import org.bson.BsonString;
+import org.junit.jupiter.api.Test;
+
+class AstDeleteCommandTests extends AbstractAstNodeTests {
+
+    @Test
+    void testRendering() {
+        // given
+        var collection = "books";
+        var filter = new AstFieldOperationFilter(
+                new AstFilterFieldPath("$isbn"),
+                new AstComparisonFilterOperation(EQ, new AstLiteralValue(new BsonString("978-3-16-148410-0"))));
+
+        var deleteCommand = new AstDeleteCommand(collection, filter);
+
+        // when && then
+        var expectedJson =
+                """
+                {"delete": "books", "deletes": [{"q": {"$isbn": {"$eq": "978-3-16-148410-0"}}, "limit": 0}]}\
```

Comment:
There seem to be no reason for using text blocks in `AstDeleteCommandTests`, `AstInsertCommandTests`, `AstComparisonFilterOperationTests`, `AstFieldOperationFilterTests`, as all of them contain only one. Let's just use single-line string literals?

Reply:
The reason is to avoid the cluttering escaping of double quotes. So below is the new one-liner:
```
"{\"delete\": \"books\", \"deletes\": [{\"q\": {\"isbn\": {\"$eq\": \"978-3-16-148410-0\"}}, \"limit\": 0}]}"
```
for simple case, it is fine.

Reply:
I did not think about this. But now, when you presented this reason, I agree that we should use multiline blocks to avoid having to escape `"`.

Suggestion for more efficient review process: if you have an objection to a proposal, or suspect that a reviewer may have failed to consider something important when making a proposal, it is better to bring that up (for example, a as a reply in the PR conversation), and see what comes from it, rather than making a change right away, and then undoing it.

Reply:
well, tbh, for one-liner I didn't think it makes much difference, as I mentioned above. but thanks for your suggestion. will keep that in mind in future.